src/generator/arm_pollers.ts

```diff
@@ -302,7 +302,7 @@ export async function generatePollers(session: Session<CodeModel>): Promise<stri
 
 
 // Creates the content in pollers_helper.go
-export async function generatePollersHelper(session: Session<CodeModel>): Promise<string> {
```

Comment:
Doesn't this go away with your recent changes to armcore?

Reply:
Yes, but that change will be implemented in a future PR. The code I have for the armcore implementation is ready to go once we get this in.